package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "wdio-cucumber-framework",
-  "version": "0.3.1",
-  "description": "A WebdriverIO plugin. Adapter for Cucumber testing framework.",
+  "version": "1.0.0-beta.0",
```

Comment:
If all existing tests have passed I am cool to just release it as `1.0.0`. There will always be bug fixes and improvements.

Edit: unless the next release takes too long than we can release it as beta with the fork as dependency